### PR TITLE
DO NOT MERGE: Try build with correct settings on 3.x

### PR DESCRIPTION
**Pull request by jordisala1991**
Added php7.1 as default
Added symfony 2.3 and 2.7 builds

Added mongodb extension for php 7.0

### CODE REVIEW

File: .travis/install_test.sh [hunk 1]
```diff
@@ -16,8 +16,9 @@ chmod u+x "${HOME}/bin/phpunit"
 wget https://github.com/satooshi/php-coveralls/releases/download/v1.0.1/coveralls.phar --output-document="${HOME}/bin/coveralls"
 chmod u+x "${HOME}/bin/coveralls"
 
-# To be removed when this issue will be resolved: https://github.com/composer/composer/issues/5355
-if [ "${COMPOSER_FLAGS}" = '--prefer-lowest' ]; then
-    composer update --prefer-dist --no-interaction --prefer-stable --quiet
-fi
+# To be removed when this issues will be resolved:
```
Comment (greg0ire) [inline .travis/install_test.sh]: "when these issues are resolved"

File: .travis/install_test.sh [hunk 2]
```diff
@@ -16,8 +16,9 @@ chmod u+x "${HOME}/bin/phpunit"
...
+# https://github.com/composer/composer/issues/5355
+# https://github.com/composer/composer/issues/5030
+composer update --prefer-dist --no-interaction --prefer-stable --ignore-platform-reqs --quiet
```
Comment (greg0ire) [inline .travis/install_test.sh]: Why `quiet`? It would make troubleshooting problems a bit hard…

**Comment by OskarStark** [comment]
any news here @jordisala1991 ?

**Comment by jordisala1991** [comment]
This is only for showing that changing those is valid for the PR I opened on dev-kit. This will never be merge from here.

**Pull request closed without merging**